src/Bootstrap.cpp

```diff
@@ -44,92 +39,147 @@ void Bootstrap::launch() {
 }
 
 bool Bootstrap::launchMinecraftLauncher() {
-	if (auto minecraftLauncherPath = systemHelper->getRegValue(HKEY_CURRENT_USER, MC_LAUNCH_REG_PATH, MC_LAUNCH_REG_KEY); minecraftLauncherPath) {
+	const Architecture::Value hostArch = systemHelper.getHostArchitecture();
+	logger.log(L"Host archiecture: " + Architecture::AsString(hostArch));
```

Comment:
Misspelling :p

Reply:
Opps, I will open an issue to remind myself to fix this in the next version. Not a major worry as it only shows in the log file 👍